bootstrap/018_warehouse_updates.sql

```diff
@@ -67,7 +67,10 @@ BEGIN
         end if;
 
         DROP TABLE IF EXISTS RAW_WH_EVT ;
-        CREATE TABLE RAW_WH_EVT AS SELECT * FROM INTERNAL_REPORTING.CLUSTER_AND_WAREHOUSE_SESSIONS_COMPLETE_AND_DAILY WHERE filterts >= :oldest_running AND SESSION_END >= :newest_completed;
+
+        -- Generate the staging table with the enumerated list of columns to avoid a 'select *'
```

Comment:
the ctas will have the same schema as the underlying view, not sure we need this anymore